guides/index.md

```diff
@@ -11,37 +11,43 @@ for information on the latest releases and new features.
+For new users, the recommended learning path is to first do the Ember.js [Quickstart](https://guides.emberjs.com/release/getting-started/quick-start/) and [Tutorial](https://guides.emberjs.com/release/tutorial/). These teach the commands while building a sample app. Then, review the [Basic use](./basic-use) section of this site as a reference.
+
+The CLI comes with a help system too. At any point, if you're not sure what a command does and what its options are, you can add `--help` to the command and press enter to find out more.
+
+```shell
+ember generate component --help
+```
 
 ## What are addons?
 
-There are thousands of JavaScript libraries that work great in Ember. When an [npm package](https://www.npmjs.com/) offers some Ember-specific conveniences, we call it an “addon.” Ember CLI’s addon system provides a way to create reusable units of code, share components and styling, extend the build tooling, and more — all with minimal configuration.
+There are thousands of JavaScript libraries that work great in Ember.
+
+When an [npm package](https://www.npmjs.com/) offers some Ember-specific features, we call it an "addon." An addon provides a way to write reusable code, share components and styling, extend the build tooling, and more—all with minimal configuration. You can visit [Ember Observer](https://emberobserver.com) to search addons.
 
-To view some of the most popular addons, visit [Ember Observer](https://emberobserver.com).
+You can also use npm packages that were made for the wider JavaScript community.
 
-Although Ember-specific packages are more specific, you can use your favorite npm packages directly too.
-By default, Ember apps created through Ember CLI include the `ember-auto-import` dependency, which enables importing npm packages directly. For example, if you want to use the `highcharts` library in your application,
-you can install and import it in your application without any other configuration. Be sure to visit the [ember-auto-import](https://github.com/ef4/ember-auto-import) documentation for more advanced usage!
+Since `v3.x`, Ember apps include the `ember-auto-import` dependency, which enables importing npm packages directly. For example, if you want to use `highcharts` in your application,
+you can install and import it without any other configuration. Be sure to visit the [ember-auto-import](https://github.com/ef4/ember-auto-import) documentation for more advanced usage!
 
 ## Why do we need a CLI?
 
-The Ember CLI is like a dependency packager, test runner, optimizer, and local server all rolled into one. Since all the features were built to work together, common tasks (such as upgrading the app version or deploying) can be automated with production-ready, open source plugins. The CLI is backwards-compatible with older Ember apps and maintains a six-week release schedule.
+The Ember CLI's job is to make your work easier.
```

Comment:
I liked this sentence so I moved it to the top of the subsection (previously, line 33).

Reply:
```suggestion
Ember CLI's job is to make your work easier.
```